packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -117,7 +117,7 @@ class UserMessages {
       'Android sdkmanager tool was found, but failed to run ($sdkManagerPath): "$error".\n'
       'Try re-installing or updating your Android SDK,\n'
       'visit ${_androidSdkInstallUrl(platform)} for detailed instructions.';
-  String androidSdkBuildToolsOutdated(String managerPath, int sdkMinVersion, String buildToolsMinVersion, Platform platform) =>
+  String androidSdkBuildToolsOutdated(String? managerPath, int sdkMinVersion, String buildToolsMinVersion, Platform platform) =>
```

Comment:
`managerPath` isn't even used in this string, it was removed in #65426. Is it needed for g3?  If not you can just remove it.